B827EBFFFE33B28C.json

```diff
@@ -0,0 +1,18 @@
+{ 
+	"gateway_conf": { 
+		"gateway_ID": "B827EBFFFE33B28C", 
+		"servers": [ 
+			{ 
+				"server_address": "ttn-router-us-west", 

```

Comment:
This is not a valid server address, please check the list here: https://www.thethingsnetwork.org/docs/gateways/packet-forwarder/semtech-udp.html#router-addresses

Reply:
```suggestion
				"server_address": "router.us.thethings.network", 
```

Reply:
I uploaded an updated file and then came here to comment. Found the "Insert a suggestion" option! That would have been easier.. :) Anyway, changes made. Thanks.